scripts/inference.py

```diff
@@ -27,13 +27,11 @@
 parser.add_argument(
     "--architecture",
     type=str,
-    default="llama",
```

Comment:
we might still want these as defaults? I think you removed this because we were considering making the parameters optional, but I think we left them as required

Reply:
reverted in latest commit